lib/actions/model.js

```diff
@@ -108,4 +108,11 @@ module.exports = {
       return validationErrors;
     };
   },
+
+  set(model, ...args) {
+    return () => {
+      model.set(...args);
```

Comment:
I think this sounds like two different jobs. One is given that I have a persisted brainstem model in the storage manager, I want to update an attribute on that model _without_ immediately persisting it back to the server. The other is I have a new model that has never been persisted, so it doesn't have an id, and I need to set an attribute on it _also without_ immediately persisting it. Maybe the former is a `brainstem-redux` concern because it deals with the storage manager (it's also a case we haven't needed to support yet), and the other isn't because it has no interaction with the storage manager?

Does that summation make sense?